### ISSUE TITLE
Wrong auth_id

### ISSUE THREAD

**Issue by 64ghost** [Vita3K/Vita3K#2389]
After installation .pkg and decrypt content changes auth_id (in eboot.bin and .suprx modules by the address 0x80). Please return the original values auth_id so that you can play on the original console without plugins

vita3k/packages/include/packages/sce_types.h

**Comment by 64ghost** [comment]
![1](https://user-images.githubusercontent.com/77530734/222232946-e0c6b049-1971-4ada-b58b-c742120fbaed.jpg)

**Comment by 64ghost** [comment]
Yes. But it is unique for each game.Console PSVita checks this value before running

**Comment by scribam** [comment]
@64ghost is it still an issue with the latest build? It is likely solved by #3107
